packages/manager/modules/pci/src/projects/project/kubernetes/components/proxy-form/proxy-form.utils.js

```diff
@@ -0,0 +1,13 @@
+export const isTimeoutValid = (timeout) => timeout === 0 || timeout >= 5;
+
+export const durationToSeconds = (duration) =>
+  typeof duration === 'string' ? moment.duration(duration).asSeconds() : 0;
```

Comment:
if duration is either a `number` or an `object`, you return `0` instead of its value. Do you really want that logic ?

Reply:
Hello, yes, the method is called `durationToSeconds` so it returns seconds as a number, even is the input in invalid